src/filter.js

```diff
@@ -73,33 +74,93 @@ define(function (require) {
    *  }
    *  </code></div>
    */
-  p5.Filter = function(type) {
-    this.ac = p5sound.audiocontext;
-
-    this.input = this.ac.createGain();
-    this.output = this.ac.createGain();
-
-    /**
-     *  The p5.Filter is built with a
-     *  <a href="http://www.w3.org/TR/webaudio/#BiquadFilterNode">
-     *  Web Audio BiquadFilter Node</a>.
-     *  
-     *  @property biquadFilter
-     *  @type {Object}  Web Audio Delay Node
-     */
+
+
+
+<<<<<<< HEAD
```

Comment:
It looks like there are some leftover markers from an unresolved merge conflict in this file. You'll have to pick through these markers manually. This is what they mean

```
<<<<<<< HEAD
This is the what you have locally
=======
```

```
=======
This is some conflicting commit
>>>>>>> a60f1128249aabc02c3d85ff12bf5f70d344f817```
```

Make sure to remove these markers keeping only the parts you want to keep, and when that's done it'll be easier for me to continue reviewing the changes to this file.